SU2_CFD/src/solvers/CTurbSSTSolver.cpp

```diff
@@ -220,16 +229,26 @@ void CTurbSSTSolver::Postprocessing(CGeometry *geometry, CSolver **solver_contai
 
     su2double VorticityMag = GeometryToolbox::Norm(3, flowNodes->GetVorticity(iPoint));
     VorticityMag = max(VorticityMag, 1e-12); // safety against division by zero
-
-    nodes->SetBlendingFunc(iPoint, mu, dist, rho);
+    su2double StrainMag = max(nodes->GetStrainMag(iPoint), 1e-12);
+    nodes->SetBlendingFunc(iPoint, mu, dist, rho, config);
 
     su2double F2 = nodes->GetF2blending(iPoint);
 
     /*--- Compute the eddy viscosity ---*/
 
     su2double kine  = nodes->GetSolution(iPoint,0);
     su2double omega = nodes->GetSolution(iPoint,1);
-    su2double zeta  = min(1.0/omega, a1/(VorticityMag*F2));
+
+    su2double P_Base;
+
+    if (sstParsedOptions.version == SST_OPTIONS::V1994){
```

Comment:
This is a bit confusing to me.  
Where does this value come from?  In  turb_sources, P_base for both version is set using the StrainMag.

Further, shouldn't we have to include the altered P_Base for the UQ terms?  (I plan to introduce the proper vorticity and kato-launder terms here as well)

Reply:
at this location, we use P_Base (defined in the if statement) to define the turbulent viscosity, which only changes depending on sst1994 or sst2003, and not on the other submodels

Reply:
Ah, so no need to include the sub-variants.